waddrmgr/common_test.go

```diff
@@ -53,6 +53,149 @@ var (
 
 	// waddrmgrNamespaceKey is the namespace key for the waddrmgr package.
 	waddrmgrNamespaceKey = []byte("waddrmgrNamespace")
+
+	// 	expectedExternalAddrs is the list of expected external addresses
+	// 	generated from the seed
+	expectedExternalAddrs = []expectedAddr{
```

Comment:
Thanks, I've organized them as suggested.

However, I had to remove a test case because addresses in the db are keyed by hash so they are not sort-able. I think the address key should include a prefix of (account + index) to make them  sortable. We already have a ticket for re-doing address keys in #242 so I'll work on it there.